benchmarks/functions/__init__.py

```diff
@@ -47,12 +47,14 @@ def _check_format(self, outputs, grad_outputs):
         grad_outputs_is_list = isinstance(grad_outputs, (list, tuple))
         if outputs_is_list and grad_outputs_is_list:
             if len(outputs) != len(grad_outputs):
-                msg = 'Format of grad_outputs is not same as that of the output of target function'
+                msg = 'Format of grad_outputs is not same as '
```

Comment:
Invalid syntax.
correct: `msg = 'Format of grad_outputs is not same as that of the output of target function' \`

Reply:
Oh, sorry.
(BTW in Chainer/CuPy we use parenthesis for line continuation instead of `\`.)